README.md

```diff
@@ -50,10 +50,10 @@ Created by [Erik Trautman](http://www.github.com/eriktrautman)
 **Our main focus** is the cultivation and sequencing of the best free resources around the internet. We believe wholeheartedly that everything one needs to know to become employed can be found for free online. However, for the average new learner, the vast quantity \(and widely varying quality\) of resources makes it difficult to make meaningful progress.  Therefore, we have devised three basic steps to fix this.
 
 1. We have created a curriculum that attempts to find the optimal sequencing of information. This curriculum is intended to take someone from knowing literally nothing about web development to a point where they could be employed as a Junior Developer.
-2. For each topic in the curriculum, we try to find the best free resources on the internet that teach that topic.  We will often link multiple resources so as to not rely too much on a single source of information.  If good resources cannot be found, we write our own, but we definitely prefer primary links to external sites.
+2. For each topic in the curriculum, we try to find the best free resources on the internet that teach that topic.  We will often link multiple resources to not rely too much on a single source of information.  If good resources cannot be found, we write our own, but we prefer primary links to external sites.
 3. We invent and cultivate projects that give learners a chance to practice what they have learned and to integrate skills along the way. These projects increase information retention, give the learner a chance to experiment, and allow the learner to build an impressive portfolio.
 
-Additionally, we have made our learning materials completely open source.  This means that if anyone comes across a resource that is better than what is currently included in our lessons, that person is free \(and encouraged!\) to add them, which improves our curriculum over time and helps it to stay up to date.
+Additionally, we have made our learning materials a completely open source.  This means that if anyone comes across a resource that is better than what is currently included in our lessons, that person is free \(and encouraged!\) to add them, which improves our curriculum over time and helps it to stay up to date.
```

Comment:
This was correct before. There should not be an article here as `completely open source` is an adjective.

Reply:
Thank you ma'am for giving your views on the changes that I have done. I will be doing the necessary changes as soon as possible and will make a PR again.